repose-aggregator/core/repose-core-api/src/test/scala/org/openrepose/core/filter/AbstractConfiguredFilterTest.scala

```diff
@@ -101,6 +101,18 @@ class AbstractConfiguredFilterTest
       verify(configurationService).subscribeTo(anyString(), anyString(), any(classOf[URL]),
         any(classOf[UpdateListener[String]]), any(classOf[Class[String]]))
     }
+
+    it("should call doInit") {
```

Comment:
It super is. I went back and forth a bit, but ended on the thought that if these methods do nothing (with the exception of logging), then there isn't really a reason to test that nothing is being done. I suppose the argument would be that logging is something, but honestly, I don't know that it's worth proving the logging call is made. For one, proving that the log line actually appears is testing part of the logging framework. Proving only that the call is made, on the other hand, would require digging around the class a bit to mock out the logger or some such thing.

I can add a test to check that the logger logs the line if you'd like, but I feel pretty meh about it.